sdks/core/src/main/java/org/gbif/pipelines/core/parsers/grscicoll/GrscicollRecordConverter.java

```diff
@@ -0,0 +1,140 @@
+package org.gbif.pipelines.core.parsers.grscicoll;
```

Comment:
The class must be in converters package:
sdks/core/src/main/java/org/gbif/pipelines/core/converters/GrscicollRecordConverter.java